packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerDelegate.java

```diff
@@ -300,8 +304,36 @@ public boolean onActivityResult(int requestCode, int resultCode, Intent data) {
 
   private void handleChoosePictureResult(int resultCode, Intent data) {
     if (resultCode == Activity.RESULT_OK && data != null) {
-      String path = fileUtils.getPathFromUri(activity, data.getData());
-      handleResult(path);
+        //String path = fileUtils.getPathFromUri(activity, data.getData());
+
+        //creating a temp file from uri to allow reading pictures from remote sources
```

Comment:
And if it isn't remote, do we then copy the file needlessly?

Please format comments as whole sentences:
```
// Create a temporary file from ...
```